impl/http/src/main/java/io/serverlessworkflow/impl/executors/http/HttpExecutor.java

```diff
@@ -47,7 +47,7 @@
 
 public class HttpExecutor implements CallableTask<CallHTTP> {
 
-  private static final Client client = ClientBuilder.newClient();
+  public static final Client client = ClientBuilder.newClient();
```

Comment:
Does client variable needs to be public?
I did not find any place where it is used.

Reply:
it was ExpressionURISupplier, i improved that